sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -724,14 +780,18 @@ def test_uniform_targets():
     for model in models_single_task:
         for y_values in (0, 5):
             y1.fill(y_values)
-            assert_array_equal(model.fit(X_train, y1).predict(X_test), y1)
+            with ignore_warnings(category=ConvergenceWarning):
```

Comment:
since we ignore the warning all the time, we could decorate the test function as well.